keras/src/ops/linalg_test.py

```diff
@@ -411,6 +417,8 @@ def test_det(self):
             linalg.det(x)
 
     def test_eig(self):
+        if testing.jax_uses_tpu():
+            self.skipTest("Skipping test with JAX + TPU as it's not supported")
```

Comment:
Oh weird, what's the error?

Reply:
`NotImplementedError: MLIR translation rule for primitive 'eig' not found for platform tpu`